CHI_DOC/Install_macos.md

```diff
@@ -1,10 +1,144 @@
+
+#### Step 2 - Install Boost
+
+Download and extract **Boost** from https://www.boost.org.  **Boost** v1.63.0+ is supported.  Configure and install as follows:
+
+    $ ./bootstrap.sh --prefix=/Path/to/Boost
+    $ ./b2 install
+
+Once the installation is complete, set the BOOST_ROOT environment variable:
+
+    $ export BOOST_ROOT=/Path/to/Boost
+
+#### Step 4 - Install PETSc
+
+Download and extract **PETSc** from https://www.mcs.anl.gov/petsc.  **PETSc** v3.9.4 is recommended for performance reasons.  Configure and install as follows:
+
+    $ ./configure --prefix=/Path/to/PETSc \
+    --with-fortran-bindings=0 \
+    --download-hypre=1 \
+    --with-ssl=0 \
+    --with-debugging=0 \
+    --with-pic=1 \
+    --with-shared-libraries=1 \
+    --with-cc=mpicc \
+    --with-cxx=mpicxx \
+    --with-fc=mpif90 \
+    --download-fblaslapack=1 \
+    --with-cxx-dialect=C++11 \
+    CFLAGS='-fPIC' \
+    CXXFLAGS='-fPIC' \
+    FCFLAGS='-fPIC' \
+    COPTFLAGS='-O3' \
+    CXXOPTFLAGS='-O3' \
+    FOPTFLAGS='-O3'
+
+If the configuration fails consult the **PETSc** user documentation.
+
+Upon completion of the configure step, **PETSc** will provide the make command
+that you should use. Execute this command.  After a successful make, **PETSc** should indicate the command used to install the header files and libraries. Execute this command.
+
+To test whether **PETSc** has been installed correctly execute:
+
+    $ make all test
+
+Finally, set the PETSC_ROOT environment variables:
+
+    $ export PETSC_ROOT=/Path/to/PETSC
+
+#### Step 5 Install the Visualization Tool Kit
+
+Download and extract **VTK** from https://vtk.org.  **VTK** v8.2.0+ is recommended.  Configure and install as follows:
+
+    $ mkdir vtk-build
+    $ cd vtk-build
+    $ cmake -DCMAKE_INSTALL_PREFIX=/Path/to/VTK \
+    -DBUILD_SHARED_LIBS:BOOL=ON \
+    -DVTK_Group_MPI:BOOL=ON \
+    -DCMAKE_BUILD_TYPE=Release \
+    /Path/to/VTK/source
+
+    $ make
+    $ make install
+
+Set the VTK_INSTALL environment variables:
+
+    $ export VTK_INSTALL=/Path/to/VTK
+
+#### Step 6 Configure and Build ChiTech
+
+Clone the **ChiTech** repository:
+
+    $ git clone https://github.com/chi-tech/chi-tech
+
+Run the configure script:
+
+    $ cd chi-tech
+    $ ./configure.sh
+
+If all goes well, configure.sh will automatically install the following four packages:
+ - readline
+ - ncurses
+ - Lua 5.3.5
+ - Triangle 1.6
+
+If problems are encountered here please see
+[troubleshooting installation](CHI_DOC/TroubleShootingInstall.md)
+
+Once configure.sh has completed, build **ChiTech** with:
+
+    $ make
+
+#### Step 7 - Generate the documentation
+
+Google and follow online instructions for installing doxygen and lua then run:
+
+    $ cd CHI_DOC/
+    $ ./YReGenerateDocumentation.sh
+
+Open the documentation index file at CHI_DOC/HTMLdocs/html/index.html.
+___
+### Compiling with the GNU toolchain
```

Comment:
I see.  I didn't realize the existing macOS instructions were just a copy/paste of the Linux instructions (I just ignored them when I added instructions for the Apple toolchain).  Since we now have proper macOS instructions,  I'm going to delete the existing Linux instructions.  Adding instructions for GCC on macOS will require some extra attention... especially in the case where they already have the Apple toolchain installed.